### PR TITLE
Removed usage of Text in Fixture and Cell

### PR DESCRIPTION
**Pull request by marcusnaslund**
The net result is actually less code.

We're stuck with some `toText() toString()` from `DateTime` but that is fixed in another PR.

`is equal to` now finally works for `String` as well.

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Peer review @thomasfanell ?

**Comment by marcusnaslund** [comment]
Actually let me first fix `Cell` here as well.

**Comment by thomasfanell** [comment]
Ok

**Comment by marcusnaslund** [comment]
Alright, this is ready now. @thomasfanell ?

File: source/unit/Fixture.ooc [hunk 1]
```diff
@@ -83,62 +85,57 @@ Fixture: abstract class {
...
+			case IntPoint2D => (expectedText, testedText) = (expectedValue toString~intpoint2d(), testedValue toString~intpoint2d())
+			case IntPoint3D => (expectedText, testedText) = (expectedValue toString~intpoint3d(), testedValue toString~intpoint3d())
+			case FloatComplex => (expectedText, testedText) = (expectedValue toString~floatcomplex(), testedValue toString~floatcomplex())
+			case FloatMatrix => (expectedText, testedText) = (expectedValue toString~floatmatrix(), testedValue toString~floatmatrix())
+			case FloatBox2D => (expectedText, testedText) = (expectedValue toString~floatbox2d(), testedValue toString~floatbox2d())
+			case IntBox2D => (expectedText, testedText) = (expectedValue toString~intbox2d(), testedValue toString~intbox2d())
+			case FloatShell2D => (expectedText, testedText) = (expectedValue toString~floatshell2d(), testedValue toString~floatshell2d())
+			case IntShell2D => (expectedText, testedText) = (expectedValue toString~intshell2d(), testedValue toString~intshell2d())
+			case => (expectedText, testedText) = (expectedValue toString(), testedValue toString()); "oopsi" println()
```
Comment (thomasfanell) [inline source/unit/Fixture.ooc]: "oopsi"?

Comment (marcusnaslund) [inline source/unit/Fixture.ooc]: Hahahah

**Comment by marcusnaslund** [comment]
Removed my embarrasing printf debugging leftover.

**Pull request merged**